libsqlite3-sys/build.rs

```diff
@@ -185,8 +185,7 @@ mod build_bundled {
                 // Windows without `-vendored-openssl` takes this to link against a prebuilt
                 // OpenSSL lib
                 cfg.include(inc_dir.to_string_lossy().as_ref());
-                let lib = lib_dir.join("libcrypto.lib");
-                cfg.flag(lib.to_string_lossy().as_ref());
+                println!("cargo:rustc-link-lib=dylib=libcrypto");
```

Comment:
So there is no difference anymore between `is_windows` (lines 184:188) and `use_openssl` (lines 190:183) (except `=libcrypto` vs `=crypto`) ?
Could you please try to fuse them ?

Reply:
OK. changed.